src/index.ts

```diff
@@ -231,6 +232,24 @@ async function getAllowance(
 	return allowance
 }
 
+async function getApproved(
```

Comment:
Is this function only for erc721 contracts? I assume so since it has the tokenId as param. Shouldn't the name be more specific then?

Also, is the word allowance or approval? We should standardize with the other erc20 allowance function

Reply:
Correct, I was following the pattern for the ERC20 tokens where there's a getAllowance function.

For ERC20s it's getAllowance (as it's literally a spend by contract allowance).. https://github.com/OpenZeppelin/openzeppelin-contracts/blob/bd4169bb15588ade629fa75302c80f0f1818c795/contracts/token/ERC20/ERC20.sol#L118

for ERC721s it's an explicit approval for another address to transaction with the tokenID 
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/bd4169bb15588ade629fa75302c80f0f1818c795/contracts/token/ERC721/ERC721.sol#L179

Happy to be more clear and standardise the other function names if you'd prefer, would something like getAllowanceERC20 / getApprovedERC721 be better?

Reply:
got it! let's adhere to the OZ names and call them

getAllowanceERC20

getApprovedERC721

getApprovedERC1155 (bool)
setApprovalForAllERC1155